netio.c

```diff
@@ -401,9 +401,9 @@ query_done(query_t query) {
 		const char *msg = or_else(query->saf_msg, "");
 
 		if (query->saf_cond == sc_limited)
-			fprintf(stderr, "Query limited: %s\n", msg);
+			fprintf(stderr, "Database limit: %s\n", msg);
```

Comment:
Is there a chance this will break anyone's scripting that looks for "Query limited"?

Reply:
if so then they deserve to lose.